bower.json

```diff
@@ -28,10 +28,14 @@
     "bignumber.js": "1.4.1",
     "angular-re-captcha": "0.0.3",
     "angulartics": "0.17.1",
-    "URIjs": "1.14.1"
+    "URIjs": "1.14.1",
+    "underscore": "~1.7.0"
```

Comment:
What was this added from?  We use lodash in stellar-client instead of underscore, so it should not be needed, as far as I can tell.